client/src/components/SearchResult/fakeData.js

```diff
@@ -0,0 +1,11 @@
+const searchData = [
+  { id: 1, address: 'W6 Garden Cafe, London', imgSrc: 'hotel' },
```

Comment:
similarily here, `imgSrc` is not an actual source of image, but more like a type/domain of business that you need to know to display proper image. Again, a better fitted name would thus be `domain`, `businessDomain`, `businessType` etc.